### DISCUSSION TITLE
electron how setting frame ？

### DISCUSSION THREAD

**Discussion by Sumxx** [umijs/umi#11978] [General]
如何在.umirc 里修改 frame 参数？

**Comment by fz6m** [comment]
umi 只是一个 SPA ，你可以把它注入到 electron 里使用，但关于 electron 的配置需要你自己改，umi 只是 web 展示层面的东西。

**Comment by Sumxx** [comment]
是的，有看了一下生成browserwindow 那部分是umi自动生成的js，我看会读取一个Config的变量合并默认参数传递给electron，但是我没找到对应的文档讲解如何配置这个Config，再.umirc里尝试了一下没有成功，所以想问问如何正确的配置这个Config来改变初始窗口的参数，比如我就想把frame改成false去掉标题

**Reply by alrexwen** [reply]
请问您目前找到解决的方案了吗
